web/components/InputEdgeConfig.jsx

```diff
@@ -0,0 +1,239 @@
+import React from "react";
+import { Button, Select, Modal, InputNumber, Divider, Tag, Tooltip, Space } from "antd";
+import PlusOutlined from "@ant-design/icons/PlusOutlined";
+import DeleteOutlined from "@ant-design/icons/DeleteOutlined";
+import { useInstance, useInstanceConfig } from "@clusterio/web_ui";
+import { InstanceSelector } from "./InstanceSelector";
+import { direction_to_text } from "../util";
+
+function EdgeStatusTag({ edge, instanceId = null }) {
+	const [instance] = useInstance(edge.target_instance);
+	const config = useInstanceConfig(edge.target_instance);
+	const target_edge = config?.["edge_transports.internal"]?.edges?.find?.(e => e.id === edge.target_edge);
+	const [targetEdgeTargetInstance] = useInstance(target_edge?.target_instance);
+
+	if (!edge.target_edge) {
+		return <Tag>Incomplete</Tag>;
+	}
+	if (!target_edge) {
+		return <Tooltip title="Target edge not found on destination instance">
+			<Tag color="warning">Missing</Tag>
+		</Tooltip>;
+	}
+	if (instanceId !== null && target_edge.target_instance !== instanceId) {
+		// eslint-disable-next-line max-len
+		return <Tooltip title={`Edge on target instance is targetting incorrect instance (${instance.name} != ${targetEdgeTargetInstance.name})`}>
+			<Tag color="error">Inconsistent</Tag>
+		</Tooltip>;
+	}
+	if (target_edge.target_edge !== edge.id) {
+		// eslint-disable-next-line max-len
+		return <Tooltip title={`Edge on target instance does not have this edge as its target (${target_edge.target_edge})`}>
+			<Tag color="error">Inconsistent</Tag>
+		</Tooltip>;
+	}
+
+	return <Tag color="success">OK</Tag>;
+}
+
+export function InputEdgeConfig({ value, onChange }) {
+	// Shim for clusterio being inconsistent with object vs stringified object handling
+	if (typeof value === "string") {
+		value = JSON.parse(value);
+	}
+
+	const [visible, setVisible] = React.useState(false);
+	const [newValue, setNewValue] = React.useState(value);
+
+	const edges = newValue.edges || [];
+
+	function displayEdge(edge, index) {
+		return <div key={`${index} ${edge.edge?.id}`}>
+			<Divider orientation="right">
+				<Space>
+					<EdgeStatusTag
+						edge={edge}
+					// Clusterio does not currently provide a way for a config input field to access the current
+					// instances ID so I am leaving this code disabled
+					// instanceId={instanceId}
+					/>
+					<Button onClick={() => {
+						setNewValue({
+							...newValue, edges: [
+								...edges.slice(0, index),
+								...edges.slice(index + 1),
+							],
+						});
+					}}>
+						<DeleteOutlined />
```

Comment:
This would look better if it was a red button or red text similar to other "stop" or "delete" buttons.

Reply:
I suppose the `danger` attribute would be a good fit, I think we use that for all other delete buttons.

Reply:
Added the danger attribute, not entirely sure it fits the look perfectly but it makes as much sense as not having it.